bot/exts/evergreen/wikipedia.py

```diff
@@ -0,0 +1,111 @@
+import asyncio
+import datetime
+import logging
+from typing import List
+
+from discord import Color, Embed, Message
+from discord.ext import commands
+
+from bot.constants import Wikipedia
+
+log = logging.getLogger(__name__)
+
+SEARCH_API = "https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch={search_term}&format=json"
+WIKIPEDIA_URL = "https://en.wikipedia.org/wiki/{title}"
+
+
+class WikipediaCog(commands.Cog):
+    """Get info from wikipedia."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.http_session = bot.http_session
+
+    @staticmethod
+    def formatted_wiki_url(index: int, titles: str) -> str:
+        """Making formatted wikipedia link."""
+        return f'`{index}` [{titles}]({WIKIPEDIA_URL.format(title=titles.replace(" ", "_"))})'
```

Comment:
was this changed back to `titles`? It's a function that takes 1 title at the time, hence why I meant it shouldn't have a plural name for title. (titles). 
```suggestion
    @staticmethod
    def formatted_wiki_url(index: int, title: str) -> str:
        """Formating wikipedia link with index and title."""
        return f'`{index}` [{title}]({WIKIPEDIA_URL.format(title=title.replace(" ", "_"))})'
```

Reply:
Sorry I understood opposite, I have now committed the changes which u propose🙂